openshift-template.yaml

```diff
@@ -112,6 +112,10 @@ parameters:
     type: boolean
     description: download certificates with certbot during install
     default: false
+  do_upgrades:
+    type: boolean
+    description: Whether to perform upgrade as part of Ansible deployment
```

Comment:
Can we be slightly more specific here? What upgrades are we doing? In this case its Operating system package upgrades for example